uikit/poidetail/src/main/java/com/sygic/ui/view/poidetail/PoiDetailBottomDialogFragment.kt

```diff
@@ -73,15 +73,19 @@ class PoiDetailBottomDialogFragment : AppCompatDialogFragment() {
 
     override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
         return BottomSheetDialog(
-            requireContext(),
-            theme,
-            resources.getDimensionPixelSize(R.dimen.defaultPeekHeight),
+            this.requireContext(),
+            this.theme,
+            this.resources.getDimensionPixelSize(R.dimen.defaultPeekHeight),
             if (preferencesManager.showcaseAllowed) SHOWCASE_BEHAVIOR_STATE else DEFAULT_BEHAVIOR_STATE
         )
     }
 
+    /**
+     * Using requireActivity().layoutInflater instead of provided inflater from onCreateView
+     * fix DialogFragment styling problem
```

Comment:
ked maju nahlaseny bug, bolo by fajn aj link nan dat. aj ked ci to niekedy v zivote fixnu netusi asi nikto

Reply:
oki, přidám ;)